modules/getting-started/proc-audit-log-view.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+// assembly-audit-log.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-audit-log-view_{context}"]
+= Viewing audit logs in {product-short}
+
+Administrators can view, search, filter, and manage the log data from the {ocp-brand-name} web console. You can filter audit logs from other log types by using the `isAuditLog` field.
+
+.Prerequisites
+* You are logged in as an administrator in the {ocp-short} web console.
+* You have configured audit logs for {product-short} on {ocp-short}. For more information, see xref:con-audit-log-config_{context}[].
+* You have a Catalog entity that contains audit log data.
```

Comment:
[SME] can you verify the accuracy of these Prereqs?

Reply:
> You are logged in as an administrator in the {ocp-short} web console.
Since we only output audit logs to console for 1.2.2, users would only need to have access to the pod logs of the RHDH pod to view the audit logs. 

> You have configured audit logs for {product-short} on {ocp-short}. For more information, see xref:con-audit-log-config_{context}[].

For 1.2.2, audit logs are on by default and will be sent to the console. Users would not have any ability to configure them at the moment.

> You have a Catalog entity that contains audit log data.
We don't store audit log data in any catalog entities, they're solely just log entries in the console for 1.2.2.

Reply:
> Since we only output audit logs to console for 1.2.2, users would only need to have access to the pod logs of the RHDH pod to view the audit logs.
@kim-tsao can you confirm the role/perms needed to view audit logs in the OCP web console?

> For 1.2.2, audit logs are on by default and will be sent to the console. Users would not have any ability to configure them at the moment.
What does this mean for the _Configuring audit logs for RHDH on OCP_ section? Do we not even need it for 1.2.2+?

> We don't store audit log data in any catalog entities, they're solely just log entries in the console for 1.2.2.
I added this because it looked like @schultzp2020 had to complete this step before he could view audit log data in his demo - is there a different way to say this or to ensure that the user has everything they need to see audit log data?

Reply:
> What does this mean for the Configuring audit logs for RHDH on OCP section? Do we not even need it for 1.2.2+?

I don't think it makes sense to have that for 1.2.2 due to no such configurations existing.

They will exist for 1.3 with the audit logging to file/splunk though.

Reply:
> I added this because it looked like @schultzp2020 had to complete this step before he could view audit log data in his demo - is there a different way to say this or to ensure that the user has everything they need to see audit log data?

Ah he was playing around with the catalog to generate audit logs for the demo otherwise we won't see many audit logs in the logs. In reality, no additional configurations are required since it'll passively collect the audit logs as users interact with the catalog, scaffolder and RBAC plugin.

Reply:
> I added this because it looked like @schultzp2020 had to complete this step before he could view audit log data in his demo - is there a different way to say this or to ensure that the user has everything they need to see audit log data?

You can remove the last bullet, as it was only included to generate audit logs. Any changes to the catalog, scaffolder, or RBAC will create an audit log.

Reply:
> @kim-tsao can you confirm the role/perms needed to view audit logs in the OCP web console?
@linfraze it would have to be the cluster admin or the namespace admin.  Someone who has access to the namespace where the RHDH pods are running.

Reply:
@kim-tsao ok, so the user will need to be logged in as `cluster-admin` or something similar, but regardless, it is true that they need to be _logged in as an administrator in the OCP web console_, right?

Reply:
> @kim-tsao ok, so the user will need to be logged in as `cluster-admin` or something similar, but regardless, it is true that they need to be _logged in as an administrator in the OCP web console_, right?

That's correct.  It would have to be an admin that has access to the namespace under which the pods are running.